README.md

```diff
@@ -487,91 +487,122 @@ gated on a condition, like tests passing.
+### 6.1. Deploying / Promoting to `dev`, `integration`, `staging` or `prod`
 
-1) Change into the Azul project root directory: `cd azul`
+We will refer to the branch of the stage to which you are deploying as the
+**`TARGET`** branch. The branch of the stage just below will be referred to as
+the **`SOURCE`** branch.
 
-2) Run `git checkout develop` (alternatively `integration`, `staging` or `prod`)
+This cheat sheet may differ from branch to branch. Be sure to follow the cheat
+sheet in the README on the branch currently checked out.
 
-3) This cheat sheet may differ from branch to branch. Be sure to follow the
-   cheat sheet in the README on the branch currently checked out.
-
-4) Run `git status` and make sure that your working copy is clean and the
-   branch is up-to-date.
-
-5) Run `source environment`
-
-6) Run `_select dev` (alternatively `integration`, `staging` or `prod`). Except
-   for `dev` and `develop`, the branch name matches the name of the deployment.
-
-7) To ensure a consistent and up-to-date set of dependencies, run
-
-   ```
-   deactivate; rm -rf .venv 
-   python3 -m venv .venv
-   source .venv/bin/activate
-   pip install -U setuptools==40.1.0 wheel==0.32.3
-   pip install -r requirements.dev.txt
-   ```
-8) Run `python scripts/envhook.py install` if you use envhook.py
-
-9) Run `make clean`
-
-10) Run `make terraform`
-
-11) Run `make deploy`
-
-12) Invoke the health and version endpoints. Be sure to use the correct 
-    deployment name:
-
-    ```
-    http https://indexer.dev.explore.data.humancellatlas.org/version
-    http https://service.dev.explore.data.humancellatlas.org/version
-    http https://indexer.dev.explore.data.humancellatlas.org/health
-    http https://service.dev.explore.data.humancellatlas.org/health
-    ```
-
-13) Run `make tag` and the `git push …` invocation that it echoes 
-
-14) Run `make subscribe`
-
-15) Run `make reindex`
-
-### 6.2 Promoting changes
-
-For promoting `dev` to `integration` use the steps outlined below. For higher
-promotions (`integration` to `staging`, or `staging` to `prod`) change the
-source and target branches accordingly. You can and should perform steps 1
-through 7 ahead of the actual promotion time.
-
-1) Change into the Azul project root directory: `cd azul`
-
-2) Checkout the target branch: `git checkout integration`
-
-3) This cheat sheet may differ from branch to branch. Be sure to follow the
-   cheat sheet in the README on the branch currently checked out.
-
-4) Run `git status` and make sure that your working copy is clean and the
-   branch is up-to-date.
-
-5) Merge the source branch: `git merge develop` and resolve any conflicts.
-   Conflict resolution should only be necessary if cherry-picks occured on the
-   target branch.
-
-6) The merge may have affected the README. Make sure you're looking at the
-   right version.
+1) **Initial Setup** (You can skip this step if you've deployed or promoted with
```

Comment:
Wouldn't this look better if it were a standard heading?

Reply:
Maybe, but then I would want to number it like the other headings. IDK. I decided no, but if you think otherwise, then say so.

Reply:
Yes, I'd prefer `6.1.1. Initial, one-time setup`. It would also reduce the use of those ugly roman numerals.